profile/repositories-list.md

```diff
@@ -0,0 +1,147 @@
+
+
+### rQES: Low Level Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [EUDI rQES CSC library (Kotlin)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-rqes-csc-kt) |  This is a Kotlin library targeting JVM, that supports the Cloud Signature Consortium API (version 2) protocol. |
+| [EUDI rQES CSC library (Swift)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-rqes-csc-swift) |  This is a Swift library targeting JVM, that supports the Cloud Signature Consortium API (version 2) protocol. |
+
+### rQES: Core and Kit Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [EUDI Remote Qualified Electronic Signature (RQES) Core library for Android](https://github.com/eu-digital-identity-wallet/eudi-lib-android-rqes-core) |  This module provides the core functionality for the EUDI Wallet for Android, focusing on the Remote Qualified Electronic Signature (RQES) service. |
+| [EUDI Remote Qualified Electronic Signature (RQES) Core library for iOS](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-rqes-kit) |  This module provides the core functionality for the EUDI Wallet for iOS, focusing on the Remote Qualified Electronic Signature (RQES) service. |
+
+### rQES: UI Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [EUDI Remote Qualified Electronic Signature (RQES) UI library for Android](https://github.com/eu-digital-identity-wallet/eudi-lib-android-rqes-ui) |  TThis module provides the core and UI functionality for the EUDI Wallet for Android implementation, focusing on the Remote Qualified Electronic Signature (RQES) service.|
+| [EUDI Remote Qualified Electronic Signature (RQES) UI library for iOS](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-rqes-ui) |  This package provides the core and UI functionality for the EUDI Wallet for iOS implementation, focusing on the Remote Qualified Electronic Signature (RQES) service.|
+
+
+</br>
+
+## EUDI Wallet mobile native applications
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Wallet UI App (Android)](https://github.com/eu-digital-identity-wallet/eudi-app-android-wallet-ui) | Implementation of wallet UI app for Android. Currently, it also includes Demo App, demonstrating the following capabilities: Proximity presentation, Same Device Online Authentication and issuing of PID and mDL.	 |
```

Comment:
'Same Device Online Authentication' → 'Same-device Online Presentation' (to match OID4VP usage and the iOS row).

Reply:
Capitalise consistently: 'Wallet UI app' → 'wallet UI app' or 'Wallet UI App' (pick one style).